rust/flatbuffers/src/endian_scalar.rs

```diff
@@ -148,34 +149,35 @@ pub fn byte_swap_f64(x: f64) -> f64 {
 
 /// Place an EndianScalar into the provided mutable byte slice. Performs
 /// endian conversion, if necessary.
+/// # Safety
+/// Caller must ensure `s.len() > size_of::<T>()`.
```

Comment:
I think we should either list all safety requirements or none, e.g. no overlapping requirement is missing.

Reply:
Done, I couldn't think of other requirements.